boards/x230.config

```diff
@@ -4,6 +4,7 @@ CONFIG_COREBOOT_CONFIG=config/coreboot-x230.config
 CONFIG_LINUX_CONFIG=config/linux-x230.config
 
 CONFIG_CRYPTSETUP=y
+CONFIG_FLASHROM=y
```

Comment:
I really should modify the x230-flash script to use flashtool. It is so much simpler than flashrom and takes up a fraction of the space.